packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -119,7 +118,6 @@ export function Kubes(props: State) {
         <Empty
           clusterId={clusterId}
           canCreate={canCreate && !isLeafCluster}
-          onClick={() => setShowAddKube(true)}
```

Comment:
From what I see there is no way for `showAddKube` to be true. Is this intended?

Reply:
Yep you're right it was missed in this PR but it's being removed in another one :)